src/inference/include/ie/ie_plugin_config.hpp

```diff
@@ -473,6 +473,7 @@ DECLARE_CONFIG_KEY(FORCE_TBB_TERMINATE);
 
 #define DECLARE_AUTO_CONFIG_KEY(name) DECLARE_CONFIG_KEY(AUTO_##name)
 
+DECLARE_AUTO_CONFIG_KEY(AUTO_SET_TO_MULTI);
```

Comment:
remove

Reply:
Okay,  I will use a simpler method to achieve this function. Please review. Thanks.